manifests/g/Gridcoin/Client/5.4.2/Gridcoin.Client.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-10
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Gridcoin.Client
+PackageVersion: 5.4.2
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2023-03-26
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/gridcoin-community/Gridcoin-Research/releases/download/5.4.2.0/gridcoin-5.4.2-win32-setup.exe
+  InstallerSha256: B7B925B26BD2C9A3D6EB386B09EF4F58F55F17B70AA06673541F7A09681A2A3E
+- Architecture: x64
+  InstallerUrl: https://github.com/gridcoin-community/Gridcoin-Research/releases/download/5.4.2.0/gridcoin-5.4.2-win64-setup.exe
+  InstallerSha256: 56087C0CF721DEB212C9DDA6B0754547CED071CAADD40651544204441B2945A7
```

Comment:
```suggestion
  InstallerSha256: 56087C0CF721DEB212C9DDA6B0754547CED071CAADD40651544204441B2945A7
  ProductCode: Gridcoin (32-bit)
```

Reply:
Product code? Shouldn't that be the display name? Also, you flipped 32-bit and 64-bit.